lms/djangoapps/verify_student/__init__.py

```diff
@@ -0,0 +1,26 @@
+# encoding: utf-8
+"""
+toggle_switch used for toggling between ona nd off for waffle switches.
+"""
+""" Activate or deactivate a feature switch. """
+
+from waffle.models import Switch
+
+
+def toggle_switch(name, active):
```

Comment:
This is code for tests, and should be co-located with test code under the `tests` module.